samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -115,15 +127,20 @@ static void fp_discoverable_adv_timeout_fn(struct k_work *w)
 
 static void connected(struct bt_conn *conn, uint8_t err)
 {
-	k_work_cancel_delayable(&fp_discoverable_adv_timeout);
-	k_work_cancel(&bt_adv_restart);
+	int ret = k_work_cancel_delayable(&fp_discoverable_adv_timeout);
+
+	__ASSERT_NO_MSG(ret == 0);
+	ret = k_work_cancel(&bt_adv_restart);
+	__ASSERT_NO_MSG(ret == 0);
+	ARG_UNUSED(ret);
```

Comment:
I would still insist on improving the formatting of code blocks within the function. I define the code block as a few subsequent lines of code that do not have a blank line in between and are at the same scope level (e.g. API call and the return value check should belong to the single code block).

L132-135 looks like a single block of code. However, L132 assert checks the return value of the previous code block. This kind of formatting makes the code hard to read.